presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -492,6 +492,17 @@ Optimizer Properties
     for any reason a cost could not be computed, the ``ELIMINATE_CROSS_JOINS`` strategy is used. This can
     also be specified on a per-query basis using the ``join_reordering_strategy`` session property.
 
+``optimizer.max-reordered-tables``
```

Comment:
It would be more intuitive to define this limit in terms of number of joins. Otherwise, someone needs to remember that `number-of-tables=number-of-joins + 1` and that setting this value to 1 or 2 is effectively turns off join reorder.

If we keep it as the number of `tables`, we should still rename it to use a more generic term  `relations`.